Cargo.toml

```diff
@@ -114,17 +141,26 @@ rkyv = { version = "0.8.8", features = ["indexmap-2", "bytes-1"] }
 memmap2 = { version = "0.6.2" }
 toml = { version = "0.8", features = ["preserve_order"] }
 indexmap = "2"
+serde = { version = "1", features = ["derive"], default-features = false }
```

Comment:
Disabling default features on serde, disables impls for Vec and HashMap. I dont think we are using them, but we should probably keep the `std` feature enabled

Enabling dervie by default is fine, because we were doing that everywhere anyway